frontend/src/components/FamiliesGraph.vue

```diff
@@ -149,15 +149,16 @@ export default {
       return colors[baseColor][modifier]
     },
     getValuesForCharacteristic(characteristicId) {
+      const usesPercentages = "percentageValueHt" in this.diagnostic
```

Comment:
Should this be `percentageValueTotalHt` ?

Reply:
Eagle eyes :eyes:

Reply:
Fix ici : https://github.com/betagouv/ma-cantine/pull/2443/commits/db6291cce081a3b05b1256ab36a0edc0f7ffbf29